### PR TITLE
chore: release jitar 0.7.4

### LINKED ISSUE #516: Release jitar 0.7.4

### PR DESCRIPTION
**Pull request by basmasking**
Fixes #516 

Changes proposed in this pull request:
- publish latest version

@MaskingTechnology/jitar

### CODE REVIEW

**Comment by basmasking** [comment]
After merge, the tag can be created for this release

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+# v0.7.3
```
Comment (petermasking) [inline CHANGELOG.md:1]: Shouldn't this be `v0.7.4`?

**Review by petermasking — CHANGES_REQUESTED** [review]

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+To do consistent releases, we use the following steps:
+
+1. Create a new issue and branch.
+1. Update the version number for the mono repo in the `package.json` file.
```
Comment (petermasking) [inline RELEASE.md:4]:
```suggestion
1. Update the version number for the mono repo in the `lerna.json` and `package.json` file.
```

Comment (basmasking) [reply RELEASE.md:4]: Lerna updates itself and doesn't need a manual update.

**Review by petermasking — APPROVED** [review]

**Pull request merged**